proof/refine/ARM/KHeap_R.thy

```diff
@@ -4677,4 +4674,16 @@ lemma active_sc_at'_cross:
                         active_sc_def)
   done
 
+lemma is_active_sc'2_cross:
+  "\<lbrakk>(s,s') \<in> state_relation; pspace_aligned s; pspace_distinct s; is_active_sc sc_ptr s;
+    sc_at sc_ptr s\<rbrakk>
+   \<Longrightarrow> is_active_sc' sc_ptr s'"
```

Comment:
I guess it makes sense to have this lemma. Just pointing out that we have `active_sc_at'_imp_is_active_sc'` and `active_sc_at'_cross` right above here which can bridge this equality. But I am happy for this to be merged as is. What we need to do one day soon is to consolidate all the "the SC is active" variations.